aes-kw/src/kek.rs

```diff
@@ -36,6 +39,18 @@ pub const IV_LEN: usize = SEMIBLOCK_SIZE;
 /// ```
 pub const IV: [u8; IV_LEN] = [0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6];
 
+/// Alternative Initial Value constant prefix for AES-KWP as defined in
+/// RFC3394 § 3.
+///
+/// <https://datatracker.ietf.org/doc/html/rfc5649#section-3>
+///
+/// ```text
+/// The Alternative Initial Value (AIV) required by this specification is
+//  a 32-bit constant concatenated to a 32-bit MLI.  The constant is (in
+//  hexadecimal) A65959A6 and occupies the high-order half of the AIV.
+/// ```
+pub const IV_PREFIX: [u8; IV_LEN / 2] = [0xA6, 0x59, 0x59, 0xA6];
```

Comment:
Should this be renamed to `KWP_IV_PREFIX`? Probably only makes sense if we rename `IV` to `KW_IV`.

Reply:
I think `KWP_IV_PREFIX` sounds good regardless of if you rename `IV`.